.github/workflows/docs.yaml

```diff
@@ -0,0 +1,29 @@
+name: docs
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'gooddata-fdw/docs/**'
+      - 'gooddata-pandas/docs/**'
+      - 'gooddata-sdk/docs/**'
```

Comment:
W need to trigger build of documentation in the case when source-code-level documentation changes. I think this workflow reacts only on changes in docs directories, which do not necessarily hold the mentioned kind of changes

Reply:
Fixed. Documentation is built also in case of code changes. I kept separate docs workflow so that application tests are not triggered for changes applied only to documentation.